packages/react-jss/src/JssProvider.js

```diff
@@ -26,9 +33,12 @@ export default class JssProvider extends Component<Props> {
     classNamePrefix: PropTypes.string,
     disableStylesGeneration: PropTypes.bool,
     children: PropTypes.node.isRequired,
-    media: PropTypes.string
+    media: PropTypes.string,
+    idOptions: PropTypes.shape({minify: PropTypes.bool})
```

Comment:
mb we should stick with one name in both jss and react-jss, because this kind of mapping often leads to confusion

Reply:
so let's do idOptions in both cases?

Reply:
I would instead use `id` to not have duplicate `options` inside the jss options

Reply:
alright, I was not sure if this will be somehow ambiguous but I like short version more too